content/technology/Material UI/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Material UI
+description: Material UI is one of the popular react UI framework used in Altinn Studio/ Altinn Studio Apps
+weight: 100
+tags: ["tjenester 3.0", "MAterial UI"]
+---
+
+#### Material UI 
+Material UI is used in Altinn Studio as the React UI framework. MAterial UI was chosen because
+- It is based on the Google's material design. 
+- It focuses exclusively on the React library
+- Can use the readily available responsive components
+- Can override the style to customize the component to match the application requirements
+- Less time 
+- It uses JSS as a styling solution that exposes a low-level API which helps in advanced overriding and theming mechanism
+
+#### Lenker
+- [Official webpage](https://material-ui.com/)  
+- [Github](https://github.com/mui-org/material-ui) 
+
+#### React Router
+React router is a third party routing library used in Altinn Studio to trigger navigation within the application.
+
+#### Lenker
+- [Official webpage](https://reacttraining.com/react-router/web/guides/quick-start)  
```

Comment:
This link takes you to the quick-start for web. Shouldn't it be https://reacttraining.com/react-router/ ?

Reply:
Fixed